CHANGELOG.md

```diff
@@ -305,7 +321,8 @@ Only cosmetic changes.
 
 This is the first release.
 
-[Unreleased]: https://github.com/topolvm/topolvm/compare/v0.8.3...HEAD
```

Comment:
The version number will be edited during the release process, so please would not edit it in this PR yet. (Would you fix the other lines with the same pattern?)

Reply:
Sorry, I removed the description about v0.8.4.